lib/upipe-v210/upipe_v210enc.c

```diff
@@ -433,6 +433,37 @@ static int upipe_v210enc_check(struct upipe *upipe, struct uref *flow_format)
     return UBASE_ERR_NONE;
 }
 
+/** @internal @This requires a ubuf manager by proxy, and amends the flow
+ * format.
+ *
+ * @param upipe description structure of the pipe
+ * @param request description structure of the request
+ * @return an error code
+ */
+static int upipe_v210enc_amend_ubuf_mgr(struct upipe *upipe,
+                                        struct urequest *request)
+{
+    struct uref *flow_format = uref_dup(request->uref);
+    UBASE_ALLOC_RETURN(flow_format);
+
+    uint64_t align;
+    if (!ubase_check(uref_pic_flow_get_align(flow_format, &align)) || !align)
+        uref_pic_flow_set_align(flow_format, 32);
+
+    if (align % 32) {
```

Comment:
align is potentially undefined if uref_pic_flow_get_align(flow_format, &align) fails.